orderer/consensus/smartbft/verifier.go

```diff
@@ -190,11 +190,22 @@ func (v *Verifier) verifyRequest(rawRequest []byte, noConfigAllowed bool) (types
 	}
 
 	if req.chHdr.Type == int32(cb.HeaderType_CONFIG) {
-		err := v.ConfigValidator.ValidateConfig(req.envelope)
+		err = v.ConfigValidator.ValidateConfig(req.envelope)
 		if err != nil {
 			v.Logger.Errorf("Error verifying config update: %v", err)
 			return types.RequestInfo{}, err
 		}
+
```

Comment:
can't we just call the request inspector's RequestId method here instead of duplicating the code below?

Reply:
I'll try

Reply:
fix